packages/common/lib/types/CredentialIssuance.types.ts

```diff
@@ -14,6 +14,9 @@ export interface CredentialRequest {
 export interface CredentialResponse {
   credential: W3CVerifiableCredential;
   format: CredentialFormat | CredentialFormat[];
+  acceptance_token?: string;
```

Comment:
Okay this is part of the latest spec, but if you read the spec it also made the credential optional as a result of this addition. That is why it is good to have these spec texts about payloads in the interfaces.

So please make the credential optional and also check where that impacts other code

Reply:
Fixed!